inst/htmljs/animint.js

```diff
@@ -1797,8 +1797,9 @@ var animint = function (to_select, json_file) {
         mouseX = d3.event.pageX;
         mouseY = d3.event.pageY;
       }
+      var safeHtml = String(content).replace(/\n/g, '<br/>');
```

Comment:
if we do this on R instead of in JS, I think we shold be able to get line breaks in legend entries too, see test below.
<img width="1612" height="545" alt="image" src="https://github.com/user-attachments/assets/3137d6ce-d15e-4e47-9179-9aef00dddb93" />

maybe also desirable in

* axes titles
* axes tick marks
* facet titles
* geom_text label

but all of these features are too much for this PR, so I will create a new issue.